wasm/src/undefined.symbols

```diff
@@ -1,4 +1,3 @@
-opa_abort
```

Comment:
This one is now explicitly imported via the attribute here: https://github.com/open-policy-agent/opa/pull/3574/files#diff-ca08fcddef8c07d5108b9cc1c3f6587cf2b9f751fa49f9f7306ae653f7c1b0b0R11